src/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as Notification } from './notification';
```

Comment:
When importing the default export isn't it easier to do `import Notification from './notification'`?

Reply:
Sweet! 😊